### PR TITLE
Various cleanups

### PR DESCRIPTION
**Pull request by gravityrail**
- Use standard PHP tags
- Render manifest from inside theme as JSON
- Make use of wp-load portable
- Dynamic manifest icons

Note that I haven't tested running this through your Docker container, just inside VVV with current stable WordPress. YMMV.

### CODE REVIEW

**Comment by gravityrail** [comment]
I signed it!

File: src/theme/functions.php [hunk 1]
```diff
@@ -82,4 +83,75 @@ function wpb_move_comment_field_to_bottom($fields) {
...
+                  'sizes' => '192x192'
+              ),
+              array(
+                  'src' => pwa_site_icon_url( 512 ),
+                  'sizes' => '512x512'
+              )
+          );
+      }
+
+      wp_send_json( $manifest );
```
Comment (surma) [inline src/theme/functions.php]: Do we need to make sure that no other `template_redirect` hooks are run afterwards with `die` or something?

Comment (gravityrail) [reply src/theme/functions.php]: wp_send_json `die`s all by itself :)

Comment (surma) [reply src/theme/functions.php]: TIL! Thanks

**Comment by surma** [comment]
🎉

**Comment by grappler** [comment]
@surma In a MVP `require_once(dirname(__FILE__).'/../../../wp-load.php');` will work but once in the wild there are different ways of installing WordPress and so the directory structure will not be the same and the code would fail.

**Pull request merged**